src/app/tasks/project-tasks-list/project-tasks-list.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit, Inject, Input, Output, EventEmitter } from '@angular/core';
+import { taskService, groupService, gradeService } from 'src/app/ajs-upgraded-providers';
+
+@Component({
+  selector: 'df-project-tasks-list',
+  templateUrl: './project-tasks-list.component.html',
+  styleUrls: ['./project-tasks-list.component.scss'],
+})
+export class ProjectTasksListComponent implements OnInit {
+  @Input() unit: any;
+  @Input() project: any = {
+    tasks: []
+  };
+
+  @Output() private onSelect = new EventEmitter<any>();
+
+  constructor(
+    @Inject(taskService) private ts: any,
+    @Inject(groupService) private grounds: any,
+    @Inject(gradeService) private grades: any
+  ) {}
+
+  ngOnInit(): void {}
+
+  get statusText(): string {
+    return this.ts.statusText;
+  }
+
+  get hideGroupSetName(): boolean {
+    return this.unit.group_sets.length === 0;
+  }
+
+  public groupSetName(id): string {
+    return this.grounds.groupSetName(id, this.unit);
+  }
+
+  public statusClass(status) {
+    return this.ts.statusClass(status);
+  }
+
+  public taskDefinition(task) {
+    return this.unit.taskDef(task.task_definition_id)
+  }
+mifr
+  public onClickItem(data) {
+    this.onSelect.emit(data);
```

Comment:
This method is emitting the task object when a task button is clicked on, but nothing appears to happen. I can see `onSelect` is being given a `updateBurndownChart` method from inside `project-progress-dashboard`, but that method doesn't exist on the parent component.